invenio_rdm_records/fixtures/tasks.py

```diff
@@ -79,7 +78,7 @@ def _get_random_community(communities):
     id = communities[r]["id"]
     # create community owner identity
     members = current_communities.service.members
-    Member.index.refresh()
+    members.indexer.refresh()
```

Comment:
Wow, thanks for this fix 👀 
Could you please double-check if [this](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/fixtures/test_cli.py#L108) test needs the same fix? Thanks!

Reply:
There's a bunch of test cases across many modules where this change could be applied.

I can't remember when exactly and where, but I remember briefly discussing fixes for tests with @ppanero.
IIRC, we concluded that it's okay for the tests to not have knowledge about configured custom search prefixes, because such configuration just isn't set there – and if it caused issues, tests would likely be failing anyway.

Reply:
That is correct. We could try applying the change everywhere but there was no easy way of getting all the occurences. Then, as Max mentioned, we use the `indexer` due to prefix knowledge.